worker/worker_package.go

```diff
@@ -29,57 +29,28 @@ func (t *Worker) handlePackage(nodeID string) error {
 		return err
 	}
 
-	var data map[string]any
+	var data map[string][]any
 	if err := json.Unmarshal(result, &data); err != nil {
 		slog.Error(fmt.Sprintf("unmarshalled data: %#v\n", data))
 		return err
 	}
 
-	if _, ok := data[`packages`]; !ok {
-		slog.Warn(fmt.Sprint("unsupported json format for packages"))
+	if _, ok := data["packages"]; !ok {
+		slog.Warn(fmt.Sprint(`unsupported json format for packages, must be in the following format :\n
+			{ "packages": [ { "name": "foo", "version": "3.1", "arch": "amd64", }, ... ], }`))
 		return nil
```

Comment:
Don't expose the expected format in the logs, we know it already.
What would be useful for troubleshoot is the unexpected received dataset, but we can't afford logging that because it's too big.
=> Better to revert the a simple "unsupported json format for packages" message